### ISSUE TITLE
Checkbox in header missing when using dt:filterPlaceholder

### ISSUE THREAD

**Issue by damienpolegato** [dandelion/dandelion-datatables#281]
Setup a table with a checkbox in the header :
```
<thead>
    <tr>
        <th dt:filterable="false" dt:property="id" dt:sortable="false" dt:renderFunction="dt-functions#toCheckbox">
            <input id="checkall" type="checkbox" onclick=""" />
        </th>
   </tr>
</thead>
```
This works well, then add `dt:filterPlaceholder="head_after"` in the table and the checkbox disappears.

**Comment by tduchateau** [comment]
Thanks for reporting.
This issue seems to be fixed in the v1.0.0.

**Comment by damienpolegato** [comment]
Great ! Any patch I could apply to get that fix or any workaround ? 
I guess I can always add my checkbox in js after table is initialized for now.
